multimedia/views.py

```diff
@@ -10,3 +18,30 @@ def book_list(request):
 def book_detail(request, isbn13):
     book = Book.objects.get(isbn13=isbn13)
     return render(request, 'multimedia/book_detail.html', {'book': book})
+
+
+
+
+class MovieListView(TemplateView):
+    template_name = 'multimedia/movie_list.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['object_list'] = Movie.objects.all()
+        return context
+
+
+class MovieDetailView(TemplateView):
```

Comment:
In the past, i wrote the function based view. But nowadays, I always write class based views because it can be reused in many different views without repetition of code.

So far, I have no opinion whether to follow this or not.

- class based views is bloated, function based views is very simple and straight forward.
- class based views is more generic (reusable), function based views is became repetitive when the code grow.
- class based views is composed by multiple mixins, which means if you need a feature, u just need to inherit the mixin without having to rewrite the logic.
- debugging techniques for class based views is slightly different, instead of looking the issue in your code (in your function definition), you may need to see the mixin code which the class inherit from.

Here is some introduction to Django class based view: https://docs.djangoproject.com/en/1.7/topics/class-based-views/intro/